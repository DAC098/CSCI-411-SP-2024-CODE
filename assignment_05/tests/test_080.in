77 811
6
29
109
412
440
444
484
509
719
813
869
1227
1318
1587
1617
1670
1985
1993
2169
2202
2370
2835
2859
2982
2985
3017
3104
3362
3367
3390
3478
3494
3633
3705
3725
4214
4259
4302
4335
4338
4343
4463
4602
4621
4711
4793
5091
5266
5539
5639
5688
5778
5873
5982
6037
6460
6588
6729
6746
6881
6936
6938
7266
7545
7609
7709
7725
7754
8315
8524
8683
8861
8970
9289
9309
9442
9540
