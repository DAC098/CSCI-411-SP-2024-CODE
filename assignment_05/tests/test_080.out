Enter the number of hotels and the ideal number of miles to travel per day: Enter 77 hotel distances each on a separate line: Hotels to visit: 10 14 21 26 35 42 48 55 60 68 71 77 
Total penalty: 67628
